src/patternfly/components/Popover/popover.scss

```diff
@@ -67,38 +94,90 @@
     .pf-c-popover__arrow {
       bottom: 0;
       left: 50%;
-      transform: translateX(var(--pf-c-popover__arrow--m-top--TranslateX)) translateY(var(--pf-c-popover__arrow--m-top--TranslateY)) rotate(var(--pf-c-popover__arrow--m-top--Rotate));
+      transform:
+        translateX(var(--pf-c-popover__arrow--m-top--TranslateX))
+        translateY(var(--pf-c-popover__arrow--m-top--TranslateY)) rotate(var(--pf-c-popover__arrow--m-top--Rotate));
     }
   }
 
   &.pf-m-bottom {
     .pf-c-popover__arrow {
       top: 0;
       left: 50%;
-      transform: translateX(var(--pf-c-popover__arrow--m-bottom--TranslateX)) translateY(var(--pf-c-popover__arrow--m-bottom--TranslateY)) rotate(var(--pf-c-popover__arrow--m-bottom--Rotate));
+      transform:
+        translateX(var(--pf-c-popover__arrow--m-bottom--TranslateX))
+        translateY(var(--pf-c-popover__arrow--m-bottom--TranslateY))
+        rotate(var(--pf-c-popover__arrow--m-bottom--Rotate));
     }
   }
 
   &.pf-m-left {
     .pf-c-popover__arrow {
       top: 50%;
       right: 0;
-      transform: translateX(var(--pf-c-popover__arrow--m-left--TranslateX)) translateY(var(--pf-c-popover__arrow--m-left--TranslateY)) rotate(var(--pf-c-popover__arrow--m-left--Rotate));
+      transform:
+        translateX(var(--pf-c-popover__arrow--m-left--TranslateX))
+        translateY(var(--pf-c-popover__arrow--m-left--TranslateY)) rotate(var(--pf-c-popover__arrow--m-left--Rotate));
     }
   }
 
   &.pf-m-right {
     .pf-c-popover__arrow {
       top: 50%;
       left: 0;
-      transform: translateX(var(--pf-c-popover__arrow--m-right--TranslateX)) translateY(var(--pf-c-popover__arrow--m-right--TranslateY)) rotate(var(--pf-c-popover__arrow--m-right--Rotate));
+      transform:
+        translateX(var(--pf-c-popover__arrow--m-right--TranslateX))
+        translateY(var(--pf-c-popover__arrow--m-right--TranslateY)) rotate(var(--pf-c-popover__arrow--m-right--Rotate));
+    }
+  }
+
+  &.pf-m-success {
+    --pf-c-popover__icon--Color: var(--pf-c-popover--m-success__icon--Color);
+
+    .pf-c-title {
+      margin-left: var(--pf-global--spacer--xl);
+    }
+  }
+
+  &.pf-m-error {
+    --pf-c-popover__icon--Color: var(--pf-c-popover--m-error__icon--Color);
+
+    .pf-c-title {
+      margin-left: var(--pf-global--spacer--xl);
     }
   }
+
+  &.pf-m-warning {
+    --pf-c-popover__icon--Color: var(--pf-c-popover--m-warning__icon--Color);
+
+    .pf-c-title {
+      margin-left: var(--pf-global--spacer--xl);
+    }
+  }
+
+  &.pf-m-info {
+    --pf-c-popover__icon--Color: var(--pf-c-popover--m-info__icon--Color);
+
+    .pf-c-title {
+      margin-left: var(--pf-global--spacer--xl);
+    }
+  }
+
+  .pf-c-popover__icon {
+    position: absolute;
+    right: var(--pf-c-popover--c-button--Left);
+    display: flex;
+    margin-top: var(--pf-c-popover__icon--MarginTop);
```

Comment:
You shouldn't need this if you move the icon in the title.

Reply:
This is not applicable anymore as I used modalBox component to reimplement the popover.